.eslintrc.js

```diff
@@ -26,21 +26,23 @@ module.exports = {
     },
   },
   rules: {
+    'jest/no-conditional-expect': 'off',
+    // 'jest/valid-title': 'off', // can't detect titles that are string variables
```

Comment:
Do we need to leave the double comment here or can we just rm the line entirely

Reply:
👍